Sources/Sentry/SentryDebugImageProvider.m

```diff
@@ -39,6 +37,21 @@ - (instancetype)initWithBinaryImageProvider:(id<SentryCrashBinaryImageProvider>)
     return self;
 }
 
+- (NSArray<SentryDebugMeta *> *)getDebugImagesForAddresses:(NSArray<NSString *> *)addresses
+{
+    NSMutableArray<SentryDebugMeta *> *result = [NSMutableArray new];
+
+    NSArray<SentryDebugMeta *> *binaryImages = [self getDebugImages];
+
+    for (SentryDebugMeta *sourceImage in binaryImages) {
+        if ([addresses containsObject:sourceImage.imageAddress]) {
```

Comment:
`m`: When using an `NSSet` for `addresses` we improve this from `n*m` to `n`.

Reply:
Why did you mark that as resolved when the issue is not addressed?

Reply:
Is the way I keep track of things I have done, even tho I haven't commit the fix.

Reply:
Please commit the fix soon after resolving comments to avoid confusion.